ant/libmoai-clean.sh

```diff
@@ -3,4 +3,4 @@ set -e # exit on error
 rm -rf libmoai/obj
 rm -rf libmoai/libs
 
-../util/pito ant-libmoai
```

Comment:
That's what I thought - anyway I fixed it and merged yesterday, as you no doubt have noticed.  Moving on ..